<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sia</class>
 <widget class="QMainWindow" name="Sia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sia</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>10</y>
      <width>571</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #B2DFDB;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>496</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="16" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>angle_y</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>angle_x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>d</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="plate_d"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="plate_h"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="plate_i"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Имя параметра</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="plate_a"/>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="plate_r"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Значение параметра</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>dz</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>dx</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>dy</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="plate_k"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>l</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="plate_l"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>r</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="plate_n"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>n</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="plate_m"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>angle_z</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="plate_actor_origin_x">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="plate_actor_origin_y">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="plate_actor_origin_z">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QDoubleSpinBox" name="plate_actor_angle_x">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QDoubleSpinBox" name="plate_actor_angle_y">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QDoubleSpinBox" name="plate_actor_angle_z">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="plate_visibility">
       <property name="text">
        <string>Видимость</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="make_plate">
       <property name="text">
        <string>MakePlate</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QCheckBox" name="is_plate_right_side">
       <property name="text">
        <string>Правосторонний</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="lower_jaw_visibility">
       <property name="text">
        <string>Видимость</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="open_lower_jaw">
       <property name="text">
        <string>Загрузить нижнюю
 челюсть</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>189</width>
      <height>385</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="10" column="0">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>dy</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>dz</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>angle_x</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>angle_y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>u</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Имя параметра</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>w</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="head_s"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="head_t"/>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="head_u"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="head_v"/>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="head_w"/>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="head_x"/>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="head_y"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>dx</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>angle_z</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="head_actor_origin_x">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="head_actor_origin_y">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="head_actor_origin_z">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="head_actor_angle_x">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="head_actor_angle_y">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="head_actor_angle_z">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Значение параметра</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="make_head">
       <property name="text">
        <string>MakeHead</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QPushButton" name="make_junction">
       <property name="text">
        <string>MakeJunction</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QPushButton" name="adhering">
       <property name="text">
        <string>Adhering</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
